src/app/frontend/common/components/resourcelist/deployment/component.ts

```diff
@@ -12,41 +12,61 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import {HttpParams} from '@angular/common/http';
-import {Component, ComponentFactoryResolver, Input} from '@angular/core';
-import {Deployment, DeploymentList, Event} from '@api/backendapi';
-import {Observable} from 'rxjs/Observable';
-import {ResourceListWithStatuses} from '../../../resources/list';
-import {NotificationsService} from '../../../services/global/notifications';
-import {EndpointManager, Resource} from '../../../services/resource/endpoint';
-import {NamespacedResourceService} from '../../../services/resource/resource';
-import {MenuComponent} from '../../list/column/menu/component';
-import {ListGroupIdentifiers, ListIdentifiers} from '../groupids';
+import { HttpParams } from '@angular/common/http';
+import { Component, ComponentFactoryResolver, Input } from '@angular/core';
+import { Deployment, DeploymentList, Event } from '@api/backendapi';
+import { Observable } from 'rxjs/Observable';
+import { ResourceListWithStatuses } from '../../../resources/list';
+import { NotificationsService } from '../../../services/global/notifications';
+import { EndpointManager, Resource } from '../../../services/resource/endpoint';
+import { NamespacedResourceService } from '../../../services/resource/resource';
+import { MenuComponent } from '../../list/column/menu/component';
+import { ListGroupIdentifiers, ListIdentifiers } from '../groupids';
 
 @Component({
   selector: 'kd-deployment-list',
   templateUrl: './template.html',
 })
-export class DeploymentListComponent extends ResourceListWithStatuses<DeploymentList, Deployment> {
-  @Input() endpoint = EndpointManager.resource(Resource.deployment, true).list();
+export class DeploymentListComponent extends ResourceListWithStatuses<
+  DeploymentList,
+  Deployment
+> {
+  @Input() endpoint = EndpointManager.resource(
+    Resource.deployment,
+    true
+  ).list();
 
   constructor(
-      private readonly deployment_: NamespacedResourceService<DeploymentList>,
-      notifications: NotificationsService, resolver: ComponentFactoryResolver) {
+    private readonly deployment_: NamespacedResourceService<DeploymentList>,
+    notifications: NotificationsService,
+    resolver: ComponentFactoryResolver
+  ) {
     super('deployment', notifications, resolver);
     this.id = ListIdentifiers.deployment;
     this.groupId = ListGroupIdentifiers.workloads;
 
     // Register status icon handlers
-    this.registerBinding(this.icon.checkCircle, 'kd-success', this.isInSuccessState);
-    this.registerBinding(this.icon.timelapse, 'kd-muted', this.isInPendingState);
+    this.registerBinding(
+      this.icon.checkCircle,
+      'kd-success',
+      this.isInSuccessState
+    );
+    this.registerBinding(
+      this.icon.timelapse,
+      'kd-muted',
+      this.isInPendingState
+    );
```

Comment:
Styling is not so bad but there are a few things I don't like. For me `prettier` is using too much of new lines like in the example above. Can we limit it somehow?

Reply:
I think so too. We can set maximum length for each line to suppress adding new line, but `pettier` does not have option such as allowing to enumerate arguments on the second and subsequent lines.
I tried to set options for `prettier` to format closer as current code in subsequent commit. Please check it. 😉